sky/services/editing/ios/keyboard_impl.mm

```diff
@@ -134,17 +137,18 @@ - (void)deleteBackward {
     : binding_(this, request.Pass()), client_([[KeyboardClient alloc] init]) {}
 
 KeyboardImpl::~KeyboardImpl() {
+  [client_ hide];
```

Comment:
The parent holds a strong reference to its view. So releasing the view will not remove it from its parent. The parent will own the remaining reference.